Makefile

```diff
@@ -1,7 +1,7 @@
 ifdef VERSION
 VERSION := $(VERSION)
 else
-VERSION := master
+VERSION := main
```

Comment:
Not sure about that @clemfromspace 
What's the impact of changing the version name?

Reply:
I didn't find any in the codebase 🤔 (but found an oversight in the Makefile)